docs/guides/meta/explain.md

```diff
@@ -3,40 +3,29 @@ layout: docu
 title: "EXPLAIN: Inspect Query Plans"
 ---
 
-In order to view the query plan of a query, prepend `EXPLAIN` to a query.
-
 ```sql
 EXPLAIN SELECT * FROM tbl;
 ```
 
-By default only the final physical plan is shown. In order to see the unoptimized and optimized logical plans, change the `explain_output` setting:
+The `EXPLAIN` statement displays the physical plan, i.e., the query plan that will get executed,
+and is enabled by prepending the query with `EXPLAIN`.
+The physical plan is a tree of operators that are executed in a specific order to produce the result of the query,
+and is generated by the query optimizer.
```

Comment:
`and is generated by the query optimizer.`, let's remove this, the optimizer only rewrites the existing physical tree, which is generated by the planner.